assets/stylesheets/common/chat-message.scss

```diff
@@ -115,9 +110,8 @@
       }
     }
 
-    .tc-avatar {
+    .chat-user-avatar {
       grid-area: avatar;
-      width: 100%;
     }
 
     .chat-message-content {
```

Comment:
Testing locally, I removed these two  (.chat-user-avatar and .chat-message-content) styles and nothing changed 🤔

Reply:
Oooops I saw this after merging, feel free to try to slim down the css if you want to